distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -267,6 +272,18 @@ public void testExport() throws Exception {
     }
 
     @Test
+    public void testExportToDirectory() throws Exception {
+        resetInitialState();
+
+        String response = console.runCommand(EXPORT_COMMAND + " " + temporaryFolder.getRoot());
+
+        assertThat(String.format("Exporting current configurations to %s.",
```

Comment:
I had that originally, been going back and forth on it. I've been thinking of refactoring a bit so that the assertions could be grouped in a method that takes in the export directory. This way they could be used everywhere instead of the current way which makes the assumption that the `exportDirectory` is under `ddfHome` via the `getExportDirectory` and `getExportSubDirectory`. Something like an `assertExportContents(Path exportDirectory)`, thoughts?